README.md

```diff
@@ -200,6 +201,11 @@ This method is used to hover over an element as soon as it is present in the DOM
 This method fills an input field with a text as soon as such field is visible.
 [Example](docs/EXAMPLES.md#fillFieldWithText)
 
+### `fillFieldWithTextAndPressEnter`
```

Comment:
This part already exists later on in this file. Maybe, move it from there to here.
By the way, the description of the existing one is more informative.